File: flutter-main/flutter-main/Pattern2/p8.dart
import 'dart:io';
void main () {
        int num=4;
        int row=num;
        for (int i=1; i<=num; i++, row--) {
        	int dec=1;
                for(int j=0; j<row; j++,dec=dec+i) {
                        stdout.write ("${dec}  ");

                }
                print("");
        }
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/session5/p8.c
//            5
//         5  6
//      5  4  3
//   5  6  7  8
//5  4  3  2  1

#include<stdio.h>
void main(){
	int row;
	printf("enter row:\n");
	scanf("%d",&row);

	for(int i=1;i<=row;i++){

		for(int sp=row;sp>i;sp--){
			printf(" \t");
		}
		int num=row;
		for(int j=1;j<=i;j++){
			if(i%2==1){
				printf("%d\t",num);
				num--;
			}else{
				printf("%d\t",num);
				num++;
			}
		}
		printf("\n");
	}
}
